python_ta/reporters/node_printers.py

```diff
@@ -134,64 +134,25 @@ def render_missing_space_in_doctest(msg, _node, source_lines=None):
-        yield from render_pep8_errors_e115(msg, _node, source_lines)
-    elif "E116" in msg.msg:
-        yield from render_pep8_errors_e116(msg, _node, source_lines)
-    elif "E122" in msg.msg or "E127" in msg.msg or "E131" in msg.msg:
-        yield from render_pep8_errors_e122_and_e127_and_e131(msg, _node, source_lines)
-    elif "E124" in msg.msg:
-        yield from render_pep8_errors_e124(msg, _node, source_lines)
-    elif "E125" in msg.msg or "E129" in msg.msg:
-        yield from render_pep8_errors_e125_and_e129(msg, _node, source_lines)
-    elif "E128" in msg.msg:
-        yield from render_pep8_errors_e128(msg, _node, source_lines)
-    elif "E201" in msg.msg or "E202" in msg.msg or "E203" in msg.msg:
-        yield from render_pep8_errors_e201_e202_e203_e211(msg, _node, source_lines)
-    elif "E221" in msg.msg:
-        yield from render_pep8_errors_e221(msg, _node, source_lines)
-    elif "E222" in msg.msg:
-        yield from render_pep8_errors_e222(msg, _node, source_lines)
-    elif "E223" in msg.msg:
-        yield from render_pep8_errors_e223(msg, _node, source_lines)
-    elif "E224" in msg.msg or "E273" in msg.msg:
-        yield from render_pep8_errors_e224_and_e273(msg, _node, source_lines)
-    elif "E226" in msg.msg:
-        yield from render_pep8_errors_e226(msg, _node, source_lines)
-    elif "E227" in msg.msg:
-        yield from render_pep8_errors_e227(msg, _node, source_lines)
-    elif "E228" in msg.msg:
-        yield from render_pep8_errors_e228(msg, _node, source_lines)
-    elif "E251" in msg.msg:
-        yield from render_pep8_errors_e251(msg, _node, source_lines)
-    elif "E261" in msg.msg:
-        yield from render_pep8_errors_e261(msg, _node, source_lines)
-    elif "E262" in msg.msg:
-        yield from render_pep8_errors_e262(msg, _node, source_lines)
-    elif "E265" in msg.msg:
-        yield from render_pep8_errors_e265(msg, _node, source_lines)
-    elif "E266" in msg.msg:
-        yield from render_pep8_errors_e266(msg, _node, source_lines)
-    elif "E272" in msg.msg:
-        yield from render_pep8_errors_e272(msg, _node, source_lines)
-    elif "E275" in msg.msg:
-        yield from render_pep8_errors_e275(msg, _node, source_lines)
-    elif "E301" in msg.msg:
-        yield from render_pep8_errors_e301(msg, _node, source_lines)
-    elif "E302" in msg.msg:
-        yield from render_pep8_errors_e302(msg, _node, source_lines)
-    elif "E303" in msg.msg:
-        yield from render_pep8_errors_e303(msg, _node, source_lines)
-    elif "E304" in msg.msg:
-        yield from render_pep8_errors_e304(msg, _node, source_lines)
-    elif "E305" in msg.msg:
-        yield from render_pep8_errors_e305(msg, _node, source_lines)
-    elif "E306" in msg.msg:
-        yield from render_pep8_errors_e306(msg, _node, source_lines)
-    else:
+    # Extract the error text from msg (denoted in tests as both msg.message and msg.msg)
+    raw_msg = getattr(msg, "msg", None) or getattr(msg, "message", None)
```

Comment:
This really shouldn't be necessary. What you can do in this case is (1) revert this change to just use `getattr(msg, "msg", "")`; and (2) if you believe there is a bug, provide a test case illustrating the bug.